resources/gcp/environment.go

```diff
@@ -37,7 +37,7 @@ type Environment struct {
 }
 
 var _ config.Env = (*Environment)(nil)
-var pulumiEnvVariables = []string{"GOOGLE_CREDENTIALS"}
+var pulumiEnvVariables = []string{"GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_CREDENTIALS"}
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
`GOOGLE_CREDENTIALS` is not recognized by `gcloud`
```suggestion
var pulumiEnvVariables = []string{"GOOGLE_APPLICATION_CREDENTIALS"}
```